.github/workflows/haskell.yml

```diff
@@ -96,6 +98,23 @@ jobs:
       if: runner.os == 'macOS'
       run: brew install autoconf automake libtool
 
+    - name: "WINDOWS: Install blst"
+      if: runner.os == 'Windows'
+      env:
+        MSYS2_PATH_TYPE: inherit
+        MSYSTEM: MINGW64
+      run: C:\\msys64\\usr\\bin\\bash.exe .github/workflows/build-blst.bash
+
+    - name: "NON-WINDOWS: Install blst"
+      if: runner.os != 'Windows'
+      run: |
+        mkdir blst-sources && cd blst-sources
+        git clone https://github.com/supranational/blst
+        cd blst
+        git reset --hard $BLST_REF
+        ./build.sh
+        cd ../..
```

Comment:
This looks almost exactly as the `.github/workflows/build-blst.bash` script. Can we reuse it here?